src/ui/forum.html

```diff
@@ -7,8 +7,9 @@
 <head>
 
     <meta charset="utf-8" />
-    <title>Button Men Online &mdash; Forum</title>
+    <title>Button Men Online</title>
```

Comment:
Yep, you're absolutely right, and you actually said all this in your description - don't stress too much about my intermediate comments while doing a first code read, a lot of them are just notes to myself.